src/com/ableton/VirtualEnv.groovy

```diff
@@ -0,0 +1,30 @@
+package com.ableton
+
+
+class VirtualEnv implements Serializable {
+  def script
+  def destDir
+
+  VirtualEnv(script, python) {
```

Comment:
`String python`?

Reply:
Fixed. Also, I figured out the type of `script` by running `print this.getClass()` in the sandbox, and gave it a proper type as well.

Reply:
Nice!